docs/documentation_styleguide.rst

```diff
@@ -0,0 +1,234 @@
+========================
+Documentation Styleguide
+========================
+
+.. topic:: Table of contents
+
+    A Table of Contents that should be shown different to admonitation.
+
+.. admonition:: Description
+
+    A guide to write Documentation for Plone and for Plone Add-ons
```

Comment:
Lower case "documentation".

I don't know whether it should be "Add-ons" or "add-ons". If it is a proper noun, then capitalize, else lower-case is better.